<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addGoods</class>
 <widget class="QDialog" name="addGoods">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>160</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Товар</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Наименование:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Количество:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Единица измерения:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Цена за единицу:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="line_Name"/>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="spin_Quan"/>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="combo_ye">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QDoubleSpinBox" name="double_Cost"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="push_Ok">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>130</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Ок</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_Cancel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>130</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
